src/kelf.cpp

```diff
@@ -97,142 +98,149 @@ int Kelf::LoadKelf(const std::string &filename)
 
     if (header.Flags & 1 || header.Flags & 0xf0000 || header.BitCount != 0) {
         // TODO: check more unknown bit flags
-        printf("This file is not supported yet and looked after.\n");
+        printf(REDBOLD "This file is not supported yet and looked after.\n");
```

Comment:
Its now super hard to check where you applied formatting, and where you actually changed the code.